srtcore/core.cpp

```diff
@@ -7879,6 +7892,7 @@ int CUDT::packLostData(CPacket& w_packet, steady_clock::time_point& w_origintime
 
     while ((w_packet.m_iSeqNo = m_pSndLossList->popLostSeq()) >= 0)
     {
+        // XXX See comment at m_iSndLastDataAck field.
         const int offset = CSeqNo::seqoff(m_iSndLastDataAck, w_packet.m_iSeqNo);
```

Comment:
Do you need this comment?

Reply:
This refers to the idea of making this field managed by sender buffer - it's the thing described there. Here is one of the most important places that would be affected.